cylc/flow/suite_files.py

```diff
@@ -1066,14 +1129,15 @@ def install_workflow(flow_name=None, source=None, run_name=None,
         INSTALL_LOG.info(f"Creating symlink from {source_link}")
         source_link.symlink_to(source)
     elif source_link.exists() and (os.readlink(source_link) == str(source)):
-        INSTALL_LOG.info("Symlink from {source_link} to {source} in place.")
+        INSTALL_LOG.info(
+            f"Symlink from \"{source_link}\" to \"{source}\" in place.")
```

Comment:
Note, this is just a missing `f` for the f string from cylc install (oops)

Reply:
Dam, I thought flake8 was supposed to complain about that?

Reply:
Shame, would be nice to have an auto-check.